src/core_write.cpp

```diff
@@ -12,17 +12,20 @@
 #include <serialize.h>
 #include <streams.h>
 #include <univalue.h>
-#include <util/system.h>
 #include <util/strencodings.h>
+#include <util/system.h>
 
 UniValue ValueFromAmount(const CAmount& amount)
```

Comment:
`CAmount` is a cheaply-copied type; while refactoring `ValueFromAmount` could pass by value; same for `FormatMoney`
```suggestion
UniValue ValueFromAmount(CAmount amount)
```

Reply:
Feedback addressed :)